storage/innobase/include/log0log.h

```diff
@@ -222,7 +225,7 @@ struct log_t
   size_t write_to_log;
 
   /** Last written LSN; protected by latch */
-  lsn_t write_lsn;
+  Atomic_relaxed<lsn_t> write_lsn;
```

Comment:
Why was this change made? I think that it would be cleanest to have `log_sys.latch` cover also the cloning of the log, which would be reflected by new data members `clone_lsn` and `clone_log`, similar to `resize_lsn` and `resize_log`.